### DISCUSSION TITLE
Get User Profile Coming up Blank for Only Some Profiles

### DISCUSSION THREAD

**Discussion by PCoop001** [cjbarrie/academictwitteR#262] [Q&A]
### Please confirm the following

- [X] I have searched the existing issues
- [X] The behaviour of the program is deviated from what is described in the documentation.
- [X] I can reproduce this problem for more than one time.
- [ ] This is NOT a 3-digit error -- it does not display an error message like `something went wrong. Status code: 400.`
- [ ] This is a 3-digit error and I have consulted the [Understanding API errors](https://cran.r-project.org/web/packages/academictwitteR/vignettes/academictwitteR-errors.html) vignette and the suggestions do not help.

### Describe the bug

I have a list of user profile IDs where I am trying to collect profile information. For most of my list, I get the expected results; however, for some user profiles, I am not getting any results. When I try running get_user_profile() on these IDs individually, they are still returning back as blank. I added a few examples in the code below....why aren't these IDs returning results?

### Expected Behavior

I expect that I would get each user's profile information, but my results return blank. 

### Steps To Reproduce

`Y <- list(
795174703878942000,
854692626015993000,
702102837807996000,
1330231762543370000,
1245801375298090000,
1147586237982740000)

get_user_profile(Y, bearer_token =)'

### Environment

R version 4.1.2 (2021-11-01)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods  
[7] base     

other attached packages:
[1] readxl_1.3.1          academictwitteR_0.3.0
[3] rtweet_0.7.0         

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.7       rstudioapi_0.13  magrittr_2.0.1  
 [4] usethis_2.1.3    tidyselect_1.1.1 R6_2.5.1        
 [7] rlang_0.4.12     fansi_0.5.0      dplyr_1.0.7     
[10] httr_1.4.2       tools_4.1.2      utf8_1.2.2      
[13] cli_3.1.0        withr_2.4.3      askpass_1.1     
[16] ellipsis_0.3.2   openssl_1.4.5    tibble_3.1.6    
[19] lifecycle_1.0.1  crayon_1.4.2     purrr_0.3.4     
[22] later_1.3.0      vctrs_0.3.8      promises_1.2.0.1
[25] fs_1.5.0         curl_4.3.2       glue_1.5.0      
[28] cellranger_1.1.0 compiler_4.1.2   pillar_1.6.4    
[31] generics_0.1.1   jsonlite_1.7.2   httpuv_1.6.3    
[34] pkgconfig_2.0.3 

### Anything else?

N/A

**Comment by chainsawriot** [comment]
@PCoop001 This is probably not a bug. Instead, some user ids might return empty results because those accounts are no longer available, e.g. deleted by the users themselves or by Twitter.

**Reply by PCoop001** [reply]
@chainsawriot But these are all active accounts, so they were not deleted by the user or Twitter. Here's an example of one: 
Username=RenoHemonc, ID=795174703878942000.

**Reply by chainsawriot** [reply]
@PCoop001 Thanks for providing an example. His user id is not the same as the one you provided. An account could have changed its `user_id` for a variety of reasons.

``` r
require(academictwitteR)
#> Loading required package: academictwitteR
get_user_id("RenoHemonc")
#>           RenoHemonc 
#> "795174703878942721"
get_user_profile("795174703878942721")
#> Processing from 1 to 1
#>                                     description url protected
#> 1 Doing my part to help onc patients. + fintwit         FALSE
#>   public_metrics.followers_count public_metrics.following_count
#> 1                           2229                           2374
#>   public_metrics.tweet_count public_metrics.listed_count
#> 1                      33253                          40
#>                                                              profile_image_url
#> 1 https://pbs.twimg.com/profile_images/1469222955271950337/HNDnB_sZ_normal.jpg
#>                   id               created_at   username            name
#> 1 795174703878942721 2016-11-06T08:03:17.000Z RenoHemonc Santhosh Ambika
#>   location verified
#> 1 Reno, NV    FALSE
```

<sup>Created on 2021-12-16 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>
